.github/workflows/release.yaml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - '[0-9]+\.[0-9]+\.[0-9]+.*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Download Artifacts
+        uses: actions/download-artifact@v2
```

Comment:
If it depends on downloading artifacts, we probably need to make this job dependent of the other one?

Reply:
Yeah, I was thinking the release workflow would be triggered independently -- perhaps it's possible to also make it dependent, though I'm not sure if that's possible.

There's a lot of ways to make a release process with github actions and I haven't fully decided which one I even really like yet.  We should probably discuss this for a few minutes, as you built the previous release pipeline